docs/framework/interop/marshaling-data-with-com-interop.md

```diff
@@ -32,30 +30,30 @@ COM interop provides support for both using COM objects from managed code and ex
   
 -   [Type Library Exporter (Tlbexp.exe)](../../../docs/framework/tools/tlbexp-exe-type-library-exporter.md), which produces a COM type library from an assembly and generates a wrapper that performs marshaling during method calls.  
   
- This section describes the processes for customizing interop wrappers when you can (or must) supply the marshaler with additional type information.  
+ The following topics describe the processes for customizing interop wrappers when you can (or must) supply the marshaler with additional type information.  
   
 ## In This Section  
- [COM Data Types](http://msdn.microsoft.com/en-us/f93ae35d-a416-4218-8700-c8218cc90061)  
+ [COM Data Types](https://msdn.microsoft.com/en-us/library/sak564ww(v=vs.100).aspx)  
```

Comment:
as I just commented on the issue, these links are not part of this topic TOC any longer, I think this section should be updated to actually reflect what's on the TOC and useful links should be moved to the next section.

Reply:
@mairaw, I've updated the two sections.